### ISSUE TITLE
Code highlighter: insufficient colour contrast

### ISSUE THREAD

**Issue by saschaleib** [saschaleib/dokuwiki-template-ad-hominem#53]
Some of the default colours by the code highlighter have insufficient contrast.

Example (light mode):
![image](https://github.com/user-attachments/assets/b761adb4-f89a-4c69-b2b7-dc85d6217778)

The situation is only slightly better in dark mode:
![image](https://github.com/user-attachments/assets/22281b81-0bdf-418f-8bd5-4bca8fb7df60)

These colours are hard-coded in the DokuWiki core. As it would probably not make much sense to add them all to the configurable colours, they should be overridden in the template to work with the default colours at least.

**Comment by saschaleib** [comment]
After the changes in the pending pull request, the same elements will look like this: 
![image](https://github.com/user-attachments/assets/1ebe2527-c926-49aa-bae5-76e304068e60)

Dark mode:
![image](https://github.com/user-attachments/assets/e8f8e1f3-c516-4bda-b954-9775620e2336)

All colours have been tested to have a contrast ratio of at least 4.5:1 against the default background.

**Comment by saschaleib** [comment]
Rolled out today.
